README.md

```diff
@@ -16,72 +16,72 @@ Backed by Y Combinator
 
 ---
 
-**Fern is an open source format for defining REST APIs.**
-You can think of it like a programming language to describe
-your API: your endpoints, types, errors, and examples.
+# Fern
 
-This repository contains the **Fern compiler.** The compiler transforms the API description into useful outputs, like:
+**Fern** is an **open source toolkit** for defining, building and generating **APIs, documentation and SDKs**. Bring an existing OpenAPI definition or describe your endpoints in a Fern specification. Fern generates backend code, specifications in various formats, and client libraries with a single command.
 
-### 🌿 SDKs
+This repository contains the Fern **compiler**. The compiler transforms an API definition into various code artifacts, such as client libraries, SDKs, or even full-fledged **Express**, **Fast API** or **Spring Boot** backends.
```

Comment:
FastAPI

Reply:
> This repository contains the Fern compiler. The compiler transforms an API definition, either in Fern or OpenAPI format into various code artifacts, such as Client Libraries, SDKs, or even fully-fledged Express or FastAPI backends.

I like the original paragraph here better, i think its important to mention both the `OpenAPI and Fern format`. Since it's jus the intro, i'm okay with just listing 2 backend frameworks as examples.

Reply:
I've mixed and matched the two - how do you like it now?